wdl-grammar/src/v1/lint/one_empty_line.rs

```diff
@@ -55,24 +55,51 @@ impl<'a> Rule<&'a Pair<'a, v1::Rule>> for OneEmptyLine {
 
         let mut n_empty_lines = 0;
 
+        // This will never get used as we overwrite on first instance.
+        let mut start: Position = Position::new(
+            NonZeroUsize::try_from(1).unwrap(),
+            NonZeroUsize::try_from(1).unwrap(),
+            1,
+        );
+
         for (line_no, start_byte_no, _end_byte_no, line) in tree.as_str().lines_with_offsets() {
             if line.trim().is_empty() {
+                if n_empty_lines == 1 {
+                    start = Position::new(
+                        NonZeroUsize::try_from(line_no.get()).unwrap(),
+                        NonZeroUsize::try_from(1).unwrap(),
+                        start_byte_no,
+                    );
+                }
+
                 n_empty_lines += 1;
             } else if n_empty_lines >= 2 {
-                warnings.push_back(self.more_than_one_empty_line(Location::Position(
-                    Position::new(
+                warnings.push_back(self.more_than_one_empty_line(Location::Span {
+                    start: start.clone(),
+                    end: Position::new(
                         NonZeroUsize::try_from(line_no.get() - 1).unwrap(),
                         NonZeroUsize::try_from(1).unwrap(),
                         start_byte_no,
                     ),
-                )));
+                }));
 
                 n_empty_lines = 0;
             } else {
                 n_empty_lines = 0;
             }
         }
 
+        if n_empty_lines > 1 {
```

Comment:
Nice work catching this case, but it's already handled: https://github.com/simojoe/wdl/blob/feat/one_empty_line/wdl-grammar/src/v1/lint/newline_eof.rs#L51

I believe by committing this we would cause "conflicting" errors reporting the same span. Maybe "conflicting" isn't the right word, but you get what I mean.

I would delete this check, and then leave a comment in it's place pointing out the "missing case".
The test for the EOF case should be updated to show this rule ignores EOF newlines (with the documented assumption another rule is catching them).